calibration/NUTNRB/CGINS-NUTNRB-01107__20190215.csv

```diff
@@ -0,0 +1,8 @@
+NTR-1107,CC_lower_wavelength_limit_for_spectra_fit,217,
+NTR-1107,CC_upper_wavelength_limit_for_spectra_fit,240,
+NTR-1107,CC_wl,"[188.93, 189.74, 190.54, 191.35, 192.15, 192.96, 193.76, 194.57, 195.37, 196.18, 196.99, 197.79, 198.6, 199.4, 200.21, 201.02, 201.83, 202.63, 203.44, 204.25, 205.06, 205.86, 206.67, 207.48, 208.29, 209.1, 209.91, 210.71, 211.52, 212.33, 213.14, 213.95, 214.76, 215.57, 216.38, 217.19, 218, 218.81, 219.62, 220.43, 221.24, 222.05, 222.86, 223.67, 224.48, 225.29, 226.1, 226.91, 227.72, 228.54, 229.35, 230.16, 230.97, 231.78, 232.59, 233.4, 234.22, 235.03, 235.84, 236.65, 237.46, 238.28, 239.09, 239.9, 240.71, 241.52, 242.34, 243.15, 243.96, 244.77, 245.59, 246.4, 247.21, 248.03, 248.84, 249.65, 250.46, 251.28, 252.09, 252.9, 253.72, 254.53, 255.34, 256.16, 256.97, 257.78, 258.6, 259.41, 260.22, 261.03, 261.85, 262.66, 263.47, 264.29, 265.1, 265.91, 266.73, 267.54, 268.35, 269.17, 269.98, 270.79, 271.61, 272.42, 273.23, 274.05, 274.86, 275.67, 276.49, 277.3, 278.11, 278.93, 279.74, 280.55, 281.37, 282.18, 282.99, 283.81, 284.62, 285.43, 286.24, 287.06, 287.87, 288.68, 289.5, 290.31, 291.12, 291.93, 292.75, 293.56, 294.37, 295.18, 295.99, 296.81, 297.62, 298.43, 299.24, 300.05, 300.87, 301.68, 302.49, 303.3, 304.11, 304.92, 305.74, 306.55, 307.36, 308.17, 308.98, 309.79, 310.6, 311.41, 312.22, 313.03, 313.84, 314.65, 315.46, 316.27, 317.08, 317.89, 318.7, 319.51, 320.32, 321.13, 321.94, 322.75, 323.56, 324.37, 325.18, 325.98, 326.79, 327.6, 328.41, 329.22, 330.02, 330.83, 331.64, 332.45, 333.25, 334.06, 334.87, 335.68, 336.48, 337.29, 338.09, 338.9, 339.71, 340.51, 341.32, 342.12, 342.93, 343.73, 344.54, 345.34, 346.15, 346.95, 347.76, 348.56, 349.37, 350.17, 350.97, 351.78, 352.58, 353.38, 354.18, 354.99, 355.79, 356.59, 357.39, 358.2, 359, 359.8, 360.6, 361.4, 362.2, 363, 363.8, 364.6, 365.4, 366.2, 367, 367.8, 368.6, 369.4, 370.19, 370.99, 371.79, 372.59, 373.39, 374.18, 374.98, 375.78, 376.57, 377.37, 378.16, 378.96, 379.76, 380.55, 381.35, 382.14, 382.93, 383.73, 384.52, 385.32, 386.11, 386.9, 387.69, 388.49, 389.28, 390.07, 390.86, 391.65, 392.45, 393.24, 394.03, 394.82]",
```

Comment:
Agrees with Python calibration parser. Looks good.

Reply:
Looks good, but not on Irminger 6.